wagtail/admin/rich_text/editors/draftail/__init__.py

```diff
@@ -63,9 +64,19 @@ def format_value(self, value):
 
         return self.converter.from_database_format(value)
 
+    def get_options(self):
+        chooserUrls = {
+            "pageChooser": reverse("wagtailadmin_choose_page"),
+            "externalLinkChooser": reverse("wagtailadmin_choose_page_external_link"),
+            "emailLinkChooser": reverse("wagtailadmin_choose_page_email_link"),
+            "phoneLinkChooser": reverse("wagtailadmin_choose_page_phone_link"),
+            "anchorLinkChooser": reverse("wagtailadmin_choose_page_anchor_link"),
+        }
+        return {**self.options, "chooserUrls": chooserUrls}
+
```

Comment:
As per other comments, let's avoid this approach and keep things simple, instead each feature should pass in any chooserUrls itself.

We should probably use this function though.

```suggestion
    def get_options(self):
        return {**self.options}

```

Reply:
I removed  the function `get_options` as it not used and didn't  exist before this PR.